pkg/service_quotas/service_quotas.go

```diff
@@ -117,18 +115,24 @@ func NewServiceQuotas(region, profile string) (QuotasInterface, error) {
 	quotasService := awsservicequotas.New(awsSession, aws.NewConfig().WithRegion(region))
 	serviceQuotasChecks, otherChecks := newUsageChecks(awsSession, aws.NewConfig().WithRegion(region))
 
+	isChina := isAwsChina(region)
+	if isChina {
+		logging.Warn("AWS china currently doesn't support service quotas, disabling...")
+	}
+
 	quotas := &ServiceQuotas{
 		session:                  awsSession,
 		region:                   region,
 		quotasService:            quotasService,
 		serviceQuotasUsageChecks: serviceQuotasChecks,
+		isAwsChina:               isChina,
 		otherUsageChecks:         otherChecks,
 	}
 	return quotas, nil
 }
 
-func isValidRegion(region string) bool {
-	availableRegions := endpoints.AwsPartition().Regions()
+func isAwsChina(region string) bool {
+	availableRegions := endpoints.AwsCnPartition().Regions()
 	_, ok := availableRegions[region]
 	return ok
 }
```

Comment:
The `isValidRegion` is still useful to just validate the region string. Maybe adding the China regions here would be sufficient? eg. check if `region` is in either `endpoints.AwsPartition().Regions()` or `endpoints.AwsCnPartition().Regions()` and then have a separate check to validate if the region is in China or make that return (bool, bool) and figure out both in one go?